src/main/java/me/dinowernli/grpc/polyglot/config/ConfigurationLoader.java

```diff
@@ -137,6 +138,15 @@ private Configuration applyOverrides(Configuration configuration) {
     overrides.tlsClientOverrideAuthority()
         .ifPresent(resultBuilder.getCallConfigBuilder()::setTlsClientOverrideAuthority);
 
+    overrides.metadata().ifPresent(metadata-> {
+      for (Map.Entry<String, String> keyValue : metadata.entries().asList()) {
+        resultBuilder.getCallConfigBuilder().addMetadataBuilder()
+                .setName(keyValue.getKey())
```

Comment:
Indentation looks off (2 spaces per indent, 4 spaces for broken lines). Please fix throughout the PR